.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
         "ko": false,
         "_": false,
         "ga_track_event": false,
+        "L": true
```

Comment:
nit: we usually use trailing commas in javascript object literals so git blame is easier.  I'm actually surprised dimagimon didn't complain.